CONTRIBUTING.md

```diff
@@ -1,86 +1,98 @@
-# Want to contribute some code?
+# Contributing to Jamulus
 
-## We’d really appreciate your support!
+We’d really appreciate your support! Please ensure that you understand the following in order to keep things organized:
+
+- If a [Github issue](https://github.com/jamulussoftware/jamulus/issues) for your feature/bug fix already exists, write a message in that issue indicating that you want to work on it.
 
-- If the [Github issue](https://github.com/jamulussoftware/jamulus/issues) for your feature/bug fix already exists, write a message in that issue saying you want to work on it.
 - Otherwise, please [post on the GitHub Discussions](https://github.com/jamulussoftware/jamulus/discussions) and say that you are planning to do some coding and explain why. Then we can discuss the specification.
-- Please only start coding after we have agreed to a specification to avoid putting lots of work into something which may not get accepted later.
+- Please begin coding only after we have agreed on a specification to avoid putting a lot of effort into something that may not be accepted later.
+
 
 ## Jamulus project/source code general principles
 
-### Stability
+### 1. Stability
 
-This has been, and must continue to be, the most important requirement. It is particularly important for use cases such as the Worldjam where instability during a live performance would be unacceptable. The following two principles are designed to support this principle.
+Instabilities during live performances such as WorldJam are not acceptable. As a result, stability has been, and must continue to be the most important requirement. The following principles are designed to support this.
 
-### [Keep it Simple and Stupid](https://en.wikipedia.org/wiki/KISS_principle) and [Do One Thing and Do It Well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well)
+### 2. [Keep it Simple and Stupid](https://en.wikipedia.org/wiki/KISS_principle) and 3. [Do One Thing and Do It Well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well)
 
-If a feature or function can be achieved in another way by another system or method, then that is preferable to building that feature into Jamulus.
+If a feature or function can be accomplished in another way by another system or method, it is preferable not to build that feature into Jamulus. Rather than of implementing each and every feature as part of Jamulus, we concentrate on a stable core and implement interfaces for interaction with third-party components as needed. The [JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md) API for example, allows you to communicate with the client and server from outside the application.
```

Comment:
```suggestion
If a feature or function can be accomplished in another way by another system or method, it is preferable not to build that feature into Jamulus. Rather than implementing each and every feature as part of Jamulus, we concentrate on a stable core and implement interfaces for interaction with third-party components as needed. The [JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md) API for example, allows you to communicate with the client and server from outside the application.
```